OJP/OJP_ModesSupport.xsd

```diff
@@ -306,7 +306,7 @@
 			</xs:enumeration>
 			<xs:enumeration value="changeWithinVehicle">
 				<xs:annotation>
-					<xs:documentation>If namely trains are split, it might be necessary to be in the right part. This is indicated with this value.</xs:documentation>
+					<xs:documentation>If namely trains are split, it might be necessary to be in the right part. This is indicated with this value. This value prerefered to remainInVehicle, when it is unclear, whether the traveler is in the right part of the train. The possibility of a necessary change within a vehicle results in the usage of changeWithinVehicle.</xs:documentation>
```

Comment:
```suggestion
					<xs:documentation>If trains are split, it may be necessary to be in the correct part of the train. Its usage is preferred over remainInVehicle if it is unclear whether the traveller is in the correct part of the train. The possibility of a necessary change within a vehicle leads to the use of changeWithinVehicl</xs:documentation>
```